form.go

```diff
@@ -551,6 +568,12 @@ func (f *Form) View() string {
 
 // Run runs the form.
 func (f *Form) Run() error {
+	debugFile, err := tea.LogToFile("debug.log", "debug")
```

Comment:
I would prob hide this behind an option or env variable... otherwise, clis made with huh will be creating debug.log files in the cwd

Reply:
Yeah, will just remove this for now.